pilot/pkg/serviceregistry/kube/controller/controller_test.go

```diff
@@ -425,6 +432,8 @@ func TestController_GetPodLocality(t *testing.T) {
 				}
 			}
 		})
+
+		break
```

Comment:
looks like you are skipping tests? was this meant for debug?

Reply:
Forget to remove it, jut for debug